torch/onnx/symbolic_opset11.py

```diff
@@ -718,14 +719,14 @@ def pad(g, input, pad, mode, value):
     mode = symbolic_helper._parse_arg(mode, "s")
     if mode == "replicate":
         return replication_pad(g, input, pad)
-    elif mode == "reflect":
+    if mode == "reflect":
```

Comment:
Q: Is there a certain reason that multiple if is preferred than using elif and else branch? I saw many of your changes tending to avoid elif and else?

Reply:
The change is for clarity (reduced nested code), because the previous branch has returned: https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/no-else-return.html